packages/palette/src/elements/ProgressBar/ProgressBar.tsx

```diff
@@ -42,6 +42,7 @@ export const ProgressBar: React.FC<ProgressBarProps> = ({
         style={{
           transition,
           transform: `translateX(-${100 - percentComplete}%)`,
+          backfaceVisibility: "hidden",
```

Comment:
@dzucconi where o where do you discover all these mysterious and obscure css props?

Reply:
For whatever reason without the transition this was rendering a bit weird (like a subpixel flicker kind of thing). I never understood why this works but sometimes if you're seeing some flicker from a transform; turning off the backface-visibility fixes it. 🤷

Reply:
(Maybe doing a 3d version of the transform would be better and also solve it)